openlibrary/plugins/worksearch/subjects.py

```diff
@@ -439,6 +416,48 @@ def work_wrapper(w: dict) -> web.storage:
         )
 
 
+@dataclass
+class SubjectMeta:
+    name: str
+    key: str
+    prefix: str
+    facet: str
+    facet_key: str
+    Engine: type['SubjectEngine'] = SubjectEngine
+
+
+SUBJECTS = [
```

Comment:
SUBJECTS is kind of confusing as a var name, makes me think its a list of strings or something. This is not a blocker, though proposing a cleanup of e.g.

```suggestion
SUBJECT_META_ENGINES = [
```